### PR TITLE
Make a custom jar with classes only from project and without logback

### PR DESCRIPTION
**Pull request by thomasegense**
use maven jar builder to exclude logback.xml
Disable old pom line that generated classes.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -433,6 +433,29 @@
         </resources>
         <plugins>
 
+      <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-jar-plugin</artifactId>
```
Comment (tokee) [inline pom.xml:438]: Best to provide a version (latest is 3.3.0 and seems to work) to ensure consistent behaviour.

**Review by tokee — CHANGES_REQUESTED** [review]
Please

- Move the `maven-jar-plugin` section so that is right before or right after the `maven-war-plugin`
- Ensure that indenting matches the rest of the file
- Update CHANGELOG
- Bump version

File: pom.xml (hunk 1 above)
Comment (thomasegense) [reply pom.xml]: Agree. I will bump and release a version when it is merged into master.
I only bump and release from master branch.

**Comment by tokee** [comment]
Looks good now, thanks!

**Pull request merged**